src/test/java/org/fundacionjala/pivotal/hooks/Hooks.java

```diff
@@ -1,20 +1,56 @@
 package org.fundacionjala.pivotal.hooks;
 
+import com.jayway.restassured.response.Response;
 import cucumber.api.java.After;
-import org.fundacionjala.pivotal.core.api.APICommons;
+
 import org.fundacionjala.pivotal.core.api.APIRequestManager;
+import org.fundacionjala.pivotal.core.api.APICommons;
+import org.fundacionjala.pivotal.core.api.APIResponseName;
+
+import java.util.Map;
 
 /**
  * Hooks.java
  * Class that will have hooks for scenarios.
  */
 public class Hooks {
 
+    private final APIResponseName date;
+
+    /**
+     * Constructor.
+     * @param date object utility instance.
+     */
+    public Hooks(final APIResponseName date) {
+        this.date = date;
+    }
+
     /**
      * After hook to delete the project created in tests.
      */
     @After("@PostConditionProject")
     public void afterCreatingProject() {
-        APIRequestManager.delete(APICommons.getEndPoint());
+       // Response resaux = APICommons.getRequestResponse().get(String.format("%s%s","Project" , date.getDate()));
+      //  System.out.println(String.format("%s%s", "/projects/", APICommons.getElement(resaux, "id")));
+       // APIRequestManager.delete(String.format("%s%s", "/projects/", APICommons.getElement(resaux, "id")));
+
+
+    }
+
+    /**
+     * After hook to delete the project created in tests.
+     */
+    @After("@DeleteProject")
+    public void deleteProject() {
+        //System.out.println(APICommons.buildEndPoint("/projects/[Project.id]"));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)